imap_processing/codice/constants.py

```diff
@@ -6,8 +6,28 @@
 
 from imap_processing.codice.utils import CoDICECompression
 
+# CDF-friendly FIELDNAMES for lo-sw-species-counts data product
+LO_SW_SPECIES_FIELDNAMES = [
+    "SW - H+",
```

Comment:
What is SW? Will there ever be another identifier, or should that be dropped because we are already in the `sw-species` file?

Reply:
These names are what CoDICE wants in the CDF file, so we will need to stick with "SW" here. But I will add some clarification on what these acronyms mean ("Sunward" in this case).